CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ## [Unreleased]
 ### Added
+- Implemented anytia and pypi versioneers (for looking up the current upstream package version)
+- Added possibility to download latest build from Koji instead of building it
+- Added functional tests run from cron jobs on travis CI
```

Comment:
Well, I consider those changes as quite beneficial and I think they deserve to be mentioned. Or is it enough to mention them in the documentation only?